internal/workflow/processing.go

```diff
@@ -439,6 +440,16 @@ func (w *ProcessingWorkflow) SessionHandler(
 		return err
 	}
 
+	// Validate PREMIS.
+	activityOpts := withActivityOptsForRequest(sessCtx)
+	err := temporalsdk_workflow.ExecuteActivity(activityOpts, xmlvalidate.Name, xmlvalidate.Params{
```

Comment:
We also need to record a preservation task for this activity and persist the errors from it, to be able to report them in the API/dashboard.